sdk/core/core-tracing/lib/utils/global.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Tracer } from "../interfaces/tracer";
+
+const GLOBAL_TRACER_VERSION = 1;
+const GLOBAL_TRACER_SYMBOL = Symbol.for("@azure/core-tracing.tracerCache");
+
+export interface TracerCache {
+  version: number;
+  tracer?: Tracer;
+}
+
+let cache: TracerCache;
+
+function loadTracerCache(): void {
+  const existingCache = (global as any)[GLOBAL_TRACER_SYMBOL];
```

Comment:
since these two files are the same except for `global` and `self`, maybe it would be nice to use a `globalThis` polyfill?

Reply:
Sounds reasonable, but I'll defer to @bterlson's wisdom on that

Reply:
Rather than stress about polyfills, for now I switched things around so `global.ts` only has to concern itself with returning the global object and pulled the caching code into `cache.ts`